### PR TITLE
Extract out helpers from pages to fix bundle issue

### PR DESCRIPTION
**Pull request by orangemug**
## Description
Extracted out curriculum API helpers to fix OWA incorrectly bundling curriculum-api-2023 graphql files.

## Issue(s)

Fixes `CUR-1115`

## How to test
This should be a no-op from a user experience point of view so checking for regressions across the visualiser and docx 

1. Go to https://deploy-preview-3026--oak-web-application.netlify.thenational.academy
2. Check for regressions, nothing though have changed

## Screenshots
Before:
Screenshot 2024-11-26 at 13 35 06

After:
Screenshot 2024-11-26 at 13 34 01

### CODE REVIEW

File: src/pages/teachers/curriculum/index.tsx [hunk 1]
```diff
@@ -73,9 +72,9 @@ const CurriculumHomePage: NextPage<CurriculumHomePageProps> = (props) => {
             <Hr $color={"white"} $mb={0} />
           </Box>
           <Flex $mt={[24, 80]} $mb={[80]}>
-            <HomepageCurriculumLandingHero
+            {/* <HomepageCurriculumLandingHero
```
Comment (orangemug) [inline src/pages/teachers/curriculum/index.tsx:75]: This is commented out and needs to be added back in once bundle issues are resolved

File: src/pages/teachers/curriculum/[subjectPhaseSlug]/[tab].tsx [hunk 2]
```diff
@@ -198,13 +129,13 @@ const CurriculumInfoPage: NextPage<CurriculumInfoPageProps> = ({
         }}
         $background={"white"}
       >
-        <CurriculumHeader
+        {/* <CurriculumHeader
```
Comment (orangemug) [inline src/pages/teachers/curriculum/[subjectPhaseSlug]/[tab].tsx:132]: This need to be added back in once bundle issues are resolved.

**Review by assadk88 — APPROVED** [review]

**Review by rightsaidjames — APPROVED** [review]
Thouroughly regression tested, no issues found.

**Pull request merged**